javalib/src/main/scala/java/io/CharArrayWriter.scala

```diff
@@ -0,0 +1,84 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package java.io
+
+class CharArrayWriter(initialSize: Int)  extends Writer {
+  if (initialSize < 0) throw new IllegalArgumentException("size must be >= 0")
```

Comment:
This one survived your purge :)
```suggestion
  if (initialSize < 0)
    throw new IllegalArgumentException("size must be >= 0")
```